cogs/vxtwitconverter/info.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name" : "VxTwitConverter",
+    "author" : ["JadeRabbit#9310"],
+    "short" : "twitter embed fix when?",
+    "description" : "Converts Twitter video embeds into vxTwitter links instead",
+    "install_msg" : "Twit converter installed",
+    "requirements" : ["urlextract"],
+    "tags" : ["server"],
+    "permissions" : [],
```

Comment:
`manage_messages` permission? It's needed for Ren to suppress embeds of an existing message. See the docstring for param `suppress` in https://discordpy.readthedocs.io/en/stable/api.html?highlight=events#discord.Message.edit